CREDITS.md

```diff
@@ -190,7 +190,7 @@
 	       nr_of_measurements, combination of flags, tycho2 number)
 	    -) add all stars from Hipparcos (incl. component solutions), and
 	       tycho2+1st supplement
-	    -) reorganize the stars in several brigthness levels and
+	    -) reorganize the stars in several brigthtness levels and
```

Comment:
brightness

Reply:
Thanks. Sorry about that. I think added some superfluous whitespace as well. need to go back and fix